boot/zephyr/prj.conf

```diff
@@ -12,6 +12,7 @@ CONFIG_MBEDTLS_CFG_FILE="config-boot.h"
 CONFIG_HEAP_MEM_POOL_SIZE=16384
 
 CONFIG_FLASH=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
```

Comment:
This is option hasn't been merged into mainline Zephyr yet. Is this meant as an RFC?